landoapi/cli.py

```diff
@@ -36,10 +44,17 @@ def get_subsystems(exclude: Optional[list[Subsystem]] = None) -> list[Subsystem]
 
 
 def create_lando_api_app() -> connexion.App:
-    from landoapi.app import construct_app, load_config
+    from landoapi.app import construct_app, construct_treestatus_app, load_config
+
+    # Determine which app to construct by looking for a Treestatus specific env variable.
+    app_constructor = (
+        construct_treestatus_app
+        if os.getenv("TREESTATUS_APP") is not None
+        else construct_app
+    )
```

Comment:
Part of the reason for using an environment variable here is that the `wsgi.py` loading doesn't get access to the CLI arguments. Since we need to load the spec into the app when we run `wsgi.py`, I went with an environment variable. The PR to add this config has been landed in the upstream cloudops-infra repo, where using a separate environment variable also has the advantage of allowing us to re-use the config for the existing LandoAPI app, with an extra environment variable added to the YAML configs.